module/x/gravity/keeper/migrations.go

```diff
@@ -0,0 +1,21 @@
+package keeper
+
+import (
+	v2 "github.com/Gravity-Bridge/Gravity-Bridge/module/x/gravity/migrations/v2"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// Migrator is a struct for handling in-place store migrations.
+type Migrator struct {
+	keeper Keeper
+}
+
+// NewMigrator returns a new Migrator.
+func NewMigrator(keeper Keeper) Migrator {
+	return Migrator{keeper: keeper}
+}
+
+// Migrate1to2 migrates from consensus version 1 to 2.
+func (m Migrator) Migrate1to2(ctx sdk.Context) error {
```

Comment:
Cosmetic issue we can line up consensus versions with gravity versions, so we ave 1.1-1.5 (upcoming) this would be a migration from 4 (currently live on both ummee and Gravity Bridge chain) to 5 upcoming. Does this make sense, I'm interested in your opinion.

Reply:
Our thoughts on this: Since current consensus version for gravity module is 1, if we set new version to 5 this would cause panic when running the upgrade since not all transitions from version 1 to 5 are defined. We could make work-around by setting current version for gravity module to 4 in SetUpgradeHandler() but this isn't the prettiest solution. It looks like the intention of consensus version for module is not to follow the version of a product, since you could have some new version that doesn't introduce breaking changes in the store, and in those cases consensus version should not be incremented at all. Also, it is possible to have multiple migrate scripts with different consensus versions within the same product release (e.g., we could add our scripts to be from version 2 to 3 instead of extending your method in which case it would be 1 to 2).

Reply:
Ah I think I see the issue, we've made consensus breaking changes 5 times now (1.0-1.5) but none of those required migrations. Maybe we could put in dummy migrations for 1-4 but I don't think that's worthwhile, probably better just to link breaking consensus changes to store upgrades via comments that describe which version they match to.

Reply:
OK, so we will proceed with consensus version 2 for gravity module, as it is in this PR?

Reply:
Yeah, my 2 cents would be to keep this as 2. But going forward, always bump the `ConsensusVersion`, even if no migrations are needed.

Maybe add a simple godoc to this method that describes the situation :)